sdk/highlight-py/highlight_io/sdk.py

```diff
@@ -284,14 +285,28 @@ def log_hook(self, span: Span, record: logging.LogRecord):
             attributes["code.filepath"] = record.pathname
             attributes["code.lineno"] = record.lineno
             attributes.update(record.args or {})
+
+            message = record.getMessage()
+            try:
```

Comment:
Just to make sure, there isn't a wait that we can recognize when something is from loguru, and skip this if not?

Reply:
That's a great point. I didn't see anything obvious but I was only using print statements. Let me spend a little more time getting a debugger working and seeing if there's any data from  this point in the call stack that we can infer on.

Reply:
<img width="1070" alt="Screenshot 2023-08-04 at 3 51 38 PM" src="https://github.com/highlight/highlight/assets/58678/b7ea9003-c0c5-4bcc-a924-23771e482022">

Nothing really that helpful in this part of the call stack 😢